test/NuGet.Core.Tests/NuGet.Commands.Test/TransitiveNoWarnUtilsTests.cs

```diff
@@ -928,5 +929,184 @@ public void DependencyNodeEqualityFails_NodesHaveDifferentPackageIdsInPackageSpe
             // Assert
             seen.Count.Should().Be(2);
         }
+
+        [Theory]
+        [InlineData(null, null, true)]
+        [InlineData(null, "", true)]
+        [InlineData(null, "NU1605", false)]
+        [InlineData("", null, true)]
+        [InlineData("", "", true)]
+        [InlineData("", "NU1605", false)]
+        [InlineData("NU1605", null, true)]
+        [InlineData("NU1605", "", true)]
+        [InlineData("NU1605", "NU1605", true)]
+        [InlineData("NU1605", "NU1604", false)]
```

Comment:
Can we add multiple log code test cases?
After all this was what the original change targeted.